src/main/kotlin/ar/edu/unsam/proyecto_de_sofware/event_nexus/model/modules/base/space_reservations/SpaceReservationsModule.kt

```diff
@@ -0,0 +1,6 @@
+package ar.edu.unsam.proyecto_de_sofware.event_nexus.model.modules.base.space_reservations
+
+class SpaceReservationsModule {
```

Comment:
Implementar el AppModule
```suggestion
class SpaceReservationsModule : AppModule {
```